pandasdmx/tests/test_writer.py

```diff
@@ -54,21 +54,28 @@ def test_write_data_arguments():
     # Attributes must be a string
     with raises(TypeError):
         sdmx.to_pandas(msg, attributes=2)
+        msg.to_pandas(attributes=2)
+        
 
     # Attributes must contain only 'dgso'
     with raises(ValueError):
         sdmx.to_pandas(msg, attributes='foobarbaz')
+        msg.to_pandas(attributes='foobarbaz')
 
 
 def test_write_data(data_path):
     msg = sdmx.read_sdmx(data_path)
 
     result = sdmx.to_pandas(msg)
+    # Check if wrapper method does the same.
+    # This involves a lot of duplication. Maybe test only for one case.
```

Comment:
Agreed!

The convenience method contains no code, and only chains to the actual writer method. It is not necessary to test its return value; only that it (a) runs without exception and (b) returns a pandas object.

Instead expanding a parametrized test like this one, remove the lines, and either add to or duplicate (with appropriate modifications and comments) one of the tests further down in the file that handles a *single* message, e.g. `test_write_categoryscheme`.